CMakeLists.txt

```diff
@@ -111,10 +115,20 @@ file(GLOB CRT_JAVA_SRC
 
 # figure out the target platform info and output the lib there
 aws_detect_target_platform(TARGET_OS TARGET_ARCH)
+message(STATUS "Compiling for ${TARGET_OS}-${TARGET_ARCH}")
 set(TARGET_LIB_DIR "lib/${TARGET_OS}/${TARGET_ARCH}")
 
 # shared lib that contains the CRT and JNI bindings, to be loaded by java
 add_library(${PROJECT_NAME} SHARED ${CRT_JAVA_HEADERS} ${CRT_JAVA_SRC})
+aws_use_package(aws-c-http)
+aws_use_package(aws-c-mqtt)
+aws_use_package(aws-c-cal)
+aws_use_package(aws-c-auth)
+aws_use_package(aws-c-common)
+aws_use_package(aws-c-io)
```

Comment:
What about aws-c-compression?

Reply:
oh, this was an overreaction to not finding libcrypto, should revert